packages/now-cli/src/util/dev/server.ts

```diff
@@ -1664,11 +1664,22 @@ export default class DevServer {
       })}`
     );
 
-    const p = spawnCommand(this.devCommand, {
```

Comment:
This `spawnCommand` already has a condition for Windows https://github.com/zeit/now/blob/b0e4f2590d14e79329dbd70a6aeb2b63a244d3c6/packages/now-build-utils/src/fs/run-user-scripts.ts#L84